src/Components/ReviewFormDetails.js

```diff
@@ -47,14 +48,15 @@ const submitReview = () => {
                 gender: gender
             }
         });
-    //   setFormError('')
+    setMissingInfo('')
       clearInputs()
       console.log('ID', id)
       console.log("POST REVIEW??", clients)
 
     } else {
-    //   setFormError('Sorry, you must input all required fields!')
-    } 
+        setMissingInfo('You must input all required fields to post a review!')
```

Comment:
I can't find this setMissingInfo function. I'll take a look further in the code when I pull it down

Reply:
Nevermind, found it :p